### DISCUSSION TITLE
Windows users: Native or Windows Subsystem for Linux (WSL)?

### DISCUSSION THREAD

**Discussion by RollingStar** [beetbox/beets#5469] [Polls]
To our Windows users, what do you use to run beets? Any issues with your chosen method?

**Comment by Luunae** [comment]
Hi! Unfortunate owner of a Windows Python superfund site here! I've been fighting the PATH & adjacent problems (Think: "malcolm in the middle hal lightbulb scene") for hours trying to get it working native. Didn't even think about WSL, but I'm giving up for now and will try that whenever I come back later.
